x-pack/plugins/cloud_experiments/server/usage/register_usage_collector.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { LDClient, LDUser } from 'launchdarkly-node-server-sdk';
+import type { UsageCollectionSetup } from '@kbn/usage-collection-plugin/server';
+
+interface Usage {
+  initialized: boolean;
+  flags: Record<string, string>;
```

Comment:
Assuming all the flags are strings because `unknown` or `any` are not allowed by the schema parsing tool. Since our plan is to map them as `flattened`, it's _sort of OK_ to claim them as strings.